### PR TITLE
Add text reference styles

### PR DESCRIPTION
**Pull request by tchow8**
Adds the .text-reference and .text-reference-number styles used by Divorce Done and Submitted pages. A copy of the style from reform-pattern-library:
https://github.com/hmcts/reform-pattern-library/blob/master/app/sass/components/_components-text.scss

### CODE REVIEW

File: assets/scss/look-and-feel/text-reference.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+Text
+*/
+
+$module: "text";
```
Comment (dtaylor7) [inline assets/scss/look-and-feel/text-reference.scss]: can you put in progress list css so other projects dont have to inject new css file?

Comment (hosainnet) [reply assets/scss/look-and-feel/text-reference.scss]: It's not to do with progress list as far as I know. It's to do witth the done page?
I guess it'd also automagically be picked up by 
https://github.com/hmcts/look-and-feel/blob/master/src/sources/lookAndFeel.js#L7
So we shouldn't have to include it manually?

Comment (tchow8) [reply assets/scss/look-and-feel/text-reference.scss]: I made the requested change but Hosain has a point. Are you happy with this?

Comment (dtaylor7) [reply assets/scss/look-and-feel/text-reference.scss]: yea fair enough, thought it was todo progress-list sorry

**Pull request merged**